ansible/roles/build_image/templates/Dockerfile.j2

```diff
@@ -0,0 +1,7 @@
+FROM {{ builder.image.base }}
```

Comment:
The Docker plugin for VSCode complains about this line: 
```text
Second argument should be AS
invalid name for build stage: "}}", name can't start with a number or contain symbols
```
Not sure if this is just the plugin or if this is a real problem.

Reply:
I guess is just the plugin. I did not have such problem.